site/content/docs/main/config/api-reference.html

```diff
@@ -1917,7 +1917,7 @@ <h3 id="projectcontour.io/v1.HeaderMatchCondition">HeaderMatchCondition
 </td>
 <td>
 <p>Name is the name of the header to match against. Name is required.
-Header names are case insensitive.</p>
+Header names are case-insensitive.</p>
```

Comment:
I should call out that I don't have the 1.31 doc fixes here because I branched before that. If you're doing a running review, hopefully you're telling github which changes you've seen so that I can add in the changes for 1.31 at some point. My preference is to amend things into the corresponding commits, but I'm happy to do w/ unsquashed "fixup" commits (and not force-push) if that helps reviewers...

The way I tend to do catch ups is to rebase the spell-check + spelling branches onto a fetched upstream and repeat the workflow until I get back to 0 errors (but I do it in a separate repository to avoid people seeing until I'm ready)...